### PR TITLE
(/custom-flows/email-sms-otp) remove error handling from code examples

### PR DESCRIPTION
**Pull request by alexisintech**
as per call with @royanger

### CODE REVIEW

**Review by S3Prototype — APPROVED** [review]

File: docs/custom-flows/email-sms-otp.mdx [hunk 1]
```diff
@@ -23,8 +23,8 @@ To use SMS OTP as an authentication strategy, you first need to enable it in the
 
 1. Navigate to the Clerk Dashboard.
 1. Go to **User & Authentication > [Email, Phone, Username](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username)** in the sidebar menu.
-1. In the **Contact information** section, ensure **Phone number** is enabled. This will allow users to add phone numbers to their accounts. If you would like to use email OTP instead, you can enable the **Email address** option instead of the phone number option.
-1. In the **Username** section, ensure that **Username** is disabled, or else the `create()` method will require a username to be passed in the params. If you would like to use usernames, you must handle collecting the username in your custom flow.
+1. In the **Contact information** section, ensure *only* **Phone number** is required. Use the settings cog icon to check if a setting is required or optional. If you would like to use email OTP instead, you can enable the **Email address** option instead of the phone number option.
```
Comment (royanger) [inline docs/custom-flows/email-sms-otp.mdx:26]: These 3 bullets feel kind of cluttered and the messaging is hard to follow. I'm wondering if approaching more like this might be better? This is a super rough draft that is more POC than anything

-----

1. For the follow code example, please set SMS to enabled and required. 
2. In the **Authentication Strategies** section please set **SMS verification code** to enabled. 
3. In the settings for **Phone Number** please set **Sign-in** and **Verify at sign-up** to enabled.

If you wish to use the following code for email and OTP then follow the above steps, but enable the options for email instead of SMS.

For the following options, if the setting is enabled and required then you will either need to pass them in the `create()` step:
* Username
* Personal Information

For the following, if the setting is enabled and required you will have to handle them in a second step of your flow where you capture the information or verify the email:
* Email with OTP or Email with Link
* Password
* Passkeys
* Multifactor

**Review by royanger — COMMENTED** [review]
I still don't love the 'Enable SMS OTP' section. Here's a suggestion for that.
